shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -612,6 +612,7 @@ - (void)flagsChanged:(NSEvent*)event {
 }
 
 - (void)mouseEntered:(NSEvent*)event {
```

Comment:
You'll need to change https://github.com/flutter/engine/blob/138c91c614d742c52aa5432b4cb921f0ff9fdee2/shell/platform/darwin/macos/framework/Source/FlutterViewController.mm#L54-L55 if you're changing this logic. I was aware of this behavior, and documented it since I didn't know it could be changed.